view/frontend/templates/page/analytics.phtml

```diff
@@ -8,7 +8,21 @@ $events = $block->getEvents();
 ?>
 <script type="text/javascript">
     require(['<?php echo $url; ?>'], function(){
-        <?php foreach($events as $event) { echo $event; } ?>
+        (function() {
+            const f = function () {
+                <?php
+                foreach($events as $event) {
+                    echo html_entity_decode($event);
```

Comment:
You've added `html_entity_decode`, but it wasn't there before. Is it necessary?

Reply:
We added it recently to wordpress because of encoding issues, but i really don't remember such cases for Magento systems. Will remove it.